scripts/jira_tickets/create_compara_release_JIRA_tickets.pl

```diff
@@ -60,7 +60,7 @@ sub main {
     $division = $division ? lc $division : 'relco';
 
     # Check if the introduced/default tickets JSON file exists
-    $tickets_json = $FindBin::Bin . '/jira_recurrent_tickets.' . $division . '.json'
+    $tickets_json = $FindBin::Bin . '/../../conf/' . $division . '/jira_recurrent_tickets.json'
```

Comment:
Since this script can be moved in the future, what do you think about:
`$tickets_json = $ENV{'ENSEMBL_CVS_ROOT_DIR'} . 'ensembl-compara/conf/' . $division . '/jira_recurrent_tickets.json'`